### PR TITLE
Fix GP errors caused by out of range db types (#155)

### LINKED ISSUE #155: Error message when submitting a report
We are receiving a generic error message at time of submission and unable to submit an entry.  This error has happened with new stations and existing stations.  We've received it after filling out all fields as well as filling out the bare minimum.  I wish I had more to provide you with but it has been difficult to pinpoint the cause and has been very inconsistent.  We've tried old stations, new stations, with and without entering habitat information, clearing our browser history, etc..

errorwoerrormessage

### PR DESCRIPTION
**Pull request by steveoh**
will close #155

### CODE REVIEW

**Comment by steveoh** [comment]
there are two fields that are whole numbers and can be converted to `LONG` ¯\_(ツ)_/¯

These two tables are the only tables that had `FLOAT`'s. All of them allowed nulls and did not participate in any domains.

File: scripts/migrations/migration_155.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    arcpy.management.DeleteField(table, fields)
+
+    for field_name, alias, precision, scale in field_infos:
+        print('adding {}'.format(field_name))
+        arcpy.management.AddField(table, field_name=field_name, field_type='FLOAT', field_precision=precision, field_scale=scale, field_alias=alias)
+
+
+table_name = 'Equipment'
+field_infos = [
+    ('VOLTAGE', 'Voltage', 5, 6),  #: min 0, max 1000, steps .01
```
Comment (stdavis) [inline scripts/migrations/migration_155.py]: Shouldn't this be precision = 6 and scale = 2? Or am I crazy?

Comment (steveoh) [reply scripts/migrations/migration_155.py]: Oops

Comment (stdavis) [reply scripts/migrations/migration_155.py]: Ladies and Gents: He's human.

**Pull request merged**